app/models/assignment.rb

```diff
@@ -896,6 +900,24 @@ def can_upload_starter_code?
     false
   end
 
+  # Returns true if this is a peer review, meaning it has a parent assignment,
+  # false otherwise.
+  def is_peer_review?
+    not self.parent_assignment_id.nil?
+  end
+
+  # Checks if this assignment has any children which reference this assignment
+  def has_peer_review_assignments?
+    not Assignment.where(:parent_assignment_id => self.parent_assignment_id).empty?
+  end
+
+  # Gets a relation object that contains all of the child assignments that
+  # are to be peer reviews for this assignment (which means .empty? may be
+  # true)
+  def get_peer_review_assignments
+    Assignment.where(:parent_assignment_id => self.parent_assignment_id)
```

Comment:
See comment above. There will only be one, and it should be an association, so you can delete this method.